lib/bolt/transport/local_windows.rb

```diff
@@ -152,7 +152,7 @@ def run_task(target, task, arguments, options = {})
           interpreter = select_interpreter(script, target.options['interpreters'])
           interpreter_debug = interpreter ? " using '#{interpreter}' interpreter" : nil
           # log the arguments with sensitive data redacted, do NOT log unwrapped_arguments
-          logger.debug("Running '#{script}' with #{arguments}#{interpreter_debug}")
+          logger.debug("Running '#{script}' with #{arguments.to_json}#{interpreter_debug}")
```

Comment:
https://github.com/puppetlabs/bolt/blob/master/lib/bolt/transport/local_windows.rb#L104 I think this line as well?

Reply:
Updated.